src/vs/workbench/contrib/interactive/browser/interactive.contribution.ts

```diff
@@ -332,7 +345,7 @@ registerAction2(class extends Action2 {
 
 		const editorInput = InteractiveEditorInput.create(accessor.get(IInstantiationService), notebookUri, inputUri);
 		historyService.clearHistory(notebookUri);
-		await editorService.openEditor(editorInput, undefined, group);
+		await editorService.openEditor(editorInput, editorOptions, group);
```

Comment:
it seems the default behavior will change from Focus the newly created IW to preserve focus, which is a breaking change when the command is invoked without any argument.

Reply:
I'm updating the Jupyter extension so that when the interactive window is created via the command palette, we still activate the interactive window: https://github.com/microsoft/vscode-jupyter/pull/6597/files#diff-b17feb438ea1894f4ff5ec0c02daf1876e50980714c120a3d6fac6f1a172dbc5R125 

However we can also change the default for `preserveFocus` to `false`--I don't have strong feelings about this one other than that extensions should be able to decide whether to activate the window based on `showOptions`, which is pulled directly from `createWebviewPanel`.

Reply:
I pushed change to make sure it's false by default and also fixed a compile error.